### DISCUSSION TITLE
Feature requet: Hide "Moved quickly" logger

### DISCUSSION THREAD

**Discussion by NoltoxGit** [PurpurMC/Purpur#1044] [__Dismissed__] [answered]
Hello Purpur' community !
As added for different errors that spam the console, why not add a configuration in `purpur.yml` that would hide the error message "_Pseudo moved too quickly! X,Y,Z_".

This is my suggestion :)

**Comment by YouHaveTrouble** [comment, marked as answer]
There is no need since you can just set those in spigot.yml
```yaml
moved-wrongly-threshold: 999999
moved-too-quickly-multiplier: 999999
  ```

**Reply by NoltoxGit** [reply]
Oh! I didn't know this ! Ty :)
